testdata/book/include_loop.yml

```
@@ -0,0 +1,14 @@
+desc: Test include loop(include was not involved.)
+vars:
+  lists:
+    - 1
+    - 2
+    - 3 
+  int: 1
+steps:
+  include_not_loop:
+    include:
+      path: include_str_int.yml
+      vars:
+        a: '010000000{{ vars.int }}'
+        b: '1601{{ vars.int }}'
```

The following will be correctly recognized as characters
```suggestion
        a: '"010000000{{ vars.int }}"'
        b: '"1601{{ vars.int }}"'
```